reference/entry.json

```diff
@@ -449,6 +449,235 @@
         ]
       }
     },
+    "/event/{eventId}/major-info": {
+      "parameters": [
+        {
+          "name": "eventId",
+          "in": "path",
+          "required": true,
+          "schema": {
+            "type": "string",
+            "format": "uuid",
+            "example": "F2D5D20C-CC98-4979-9CD2-453ABAD979B5"
+          },
+          "description": "Unique id of an event, in the format of a UUID"
+        }
+      ],
+      "post": {
```

Comment:
The frontend is using the `PUT` method https://github.com/cultuurnet/udb3-angular/blob/7457c3ab99b5e8cfc2f64004bbffb850fe6d0112/src/core/udb-api.service.js#L809

Reply:
The code you linked uses the post method?

Reply:
I got that mixed up with the `PUT` and `POST` in the silex app where the deprecated controller uses `POST` and the other controller uses `PUT`

https://github.com/cultuurnet/udb3-silex/blob/a17e086a8bb394184061f30b6e8a088ce79d5aa8/app/Event/DeprecatedEventControllerProvider.php#L40

https://github.com/cultuurnet/udb3-silex/blob/a17e086a8bb394184061f30b6e8a088ce79d5aa8/app/Event/EventControllerProvider.php#L76

Reply:
Yes that's how I remembered it, because you told me the frontend even didn't use the "new" major-info endpoint yet 😄